temporalcli/commands.workflow.go

```diff
@@ -96,16 +141,29 @@ func (*TemporalWorkflowUpdateCommand) run(*CommandContext, []string) error {
 	return fmt.Errorf("TODO")
 }
 
+func defaultReason() string {
+	username := "<unknown-user>"
+	if u, err := user.Current(); err != nil && u.Username != "" {
+		username = u.Username
+	}
+	return "Requested from CLI by " + username
+}
+
+type singleOrBatchOverrides struct {
+	AllowReasonWithWorkflowID bool
+}
+
 func (s *SingleWorkflowOrBatchOptions) workflowExecOrBatch(
 	cctx *CommandContext,
 	namespace string,
 	cl client.Client,
+	overrides singleOrBatchOverrides,
```

Comment:
Probably can toss all of this in a single struct instead of keeping overrides separate, but no biggie